cmd/root.go

```diff
@@ -70,6 +71,14 @@ func newRootCommand(gs *state.GlobalState) *rootCommand {
 		rootCmd.AddCommand(sc(gs))
 	}
 
+	// hack to keep backwards compatibility with "k6 cloud" command to run a test from script
+	subCmd, _, err := rootCmd.Find(gs.CmdArgs[1:])
+	if err == nil && subCmd != nil && subCmd.Name() == "cloud" {
+		index := slices.Index(gs.CmdArgs, "cloud")
+		args := slices.Insert(gs.CmdArgs, index+1, "test", "run")[1:]
```

Comment:
This will result in `k6 cloud test run` being a result of a command lookup during execution. This leads to `k6 cloud -h` printing help for `k6 cloud test run`, but `k6 help cloud` works fine.